### PR TITLE
Fix flow/tests/lint for all packages

### PR DESCRIPTION
**Pull request by micburks**
Test PR to get all green lights in build plan

### CODE REVIEW

File: fusion-cli/build/plugins/child-compilation-plugin.js [hunk 1]
```diff
@@ -39,7 +39,8 @@ class ChildCompilationPlugin {
   }
 
   apply(compiler /*: Object*/) {
-    compiler.plugin('make', (compilation, callback) => {
+    const name = this.constructor.name;
+    compiler.hooks.make.tapAsync(name, (compilation, callback) => {
```
Comment (micburks) [inline fusion-cli/build/plugins/child-compilation-plugin.js]: This was the actual change intended for this PR

**Comment by micburks** [comment]
Removed the original PR content and renamed

File: fusion-cli/get-test-files.js [hunk 2]
```diff
@@ -28,12 +28,19 @@ function getFiles() /*: Array<string> */ {
       '**/*.test.js',
       ...testPathIgnorePatterns.map(pattern => `!${pattern}`),
     ],
-    {gitignore: true}
+    {
+      gitignore: true,
+      ignore: ['node_modules']
```
Comment (micburks) [inline fusion-cli/get-test-files.js]: This was preventing me from running this script locally

File: fusion-cli/get-test-files.js [hunk 3]
```diff
@@ -28,12 +28,19 @@ function getFiles() /*: Array<string> */ {
...
+    }
   );
 
-  const filesPerJob = Math.ceil(allFiles.length / JOB_COUNT);
+  const filesPerJob = Math.floor(allFiles.length / JOB_COUNT);
```
Comment (micburks) [inline fusion-cli/get-test-files.js]: If Math.ceil was rounding more than the number of tests per job, the last job was not being passed any paths. This was causing the final job to run all fusion-cli tests at once and timing out

**Comment by chrisdothtml** [comment]
Thanks again for fixing all this 👍

**Comment by KevinGrandon** [comment]
Awesome pull request, thanks for the work here. We have one failing test in fusion-cli after rebasing, no idea why - will figure that out in a follow-up.

**Pull request merged**